README.md

```diff
@@ -358,6 +358,30 @@ the warnings with correct source code information.
 `-fcolor-diagnostics` above instructs Clang to generate color output
 (unfortunately Markdown doesn't render the colors here).
 
+According to [Clang Plugins](https://clang.llvm.org/docs/ClangPlugins.html#using-the-clang-command-line)
+> If the plugin class implements the `getActionType` method then the plugin is run automatically. 
+```c
+// Automatically run the plugin after the main AST action
+PluginASTAction::ActionType getActionType() override {
+  return AddAfterMainAction;
+}
+```
+
+The **CodeStyleChecker** plugin could be automatically load and be used during the normal compilation 
+process to detect errors in the code, and get the output file, for example:
+```bash
+$Clang_DIR/bin/clang -fplugin=libCodeStyleChecker.dylib -o file.o -c file.cpp
+file.cpp:2:7: warning: Type and variable names should start with upper-case letter
+class clangTutor_BadName;
+      ^~~~~~~~~~~~~~~~~~~
+      ClangTutor_BadName
+file.cpp:2:17: warning: `_` in names is not allowed
+class clangTutor_BadName;
+      ~~~~~~~~~~^~~~~~~~~
+      clangTutorBadName
+2 warnings generated.
```

Comment:
Add `ls file.o` (together without output) before and after invoking `clang` to demonstrate that the file was indeed created.